azure/v11.4.0/README.md

```diff
@@ -0,0 +1,41 @@
+## :zap:  Giant Swarm Release 11.4.0 for Azure :zap:
+
+**If you are upgrading from 11.3.3, upgrading to this release will not roll your nodes. It will only update the apps.**
```

Comment:
This is false, if we're going to use azure operator 4.1.0

Reply:
4.1.0-dev is being used but I didn't observe nodes being rolled. Will test once more and pay attention to that aspect.

Reply:
Removed this note, since after all we will be upgrading k8s in the same release